src/mote/templates/teams.html

```diff
@@ -0,0 +1,380 @@
+    <link href="{{ url_for('static', filename='css3/teams.css') }}" rel="stylesheet" />
+    <title>Meetbot Logs</title>
+</head>
+
+<body class="body bg-light">
+    <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-light"
+        style="background-image: linear-gradient(180deg,#eee 0,#ddd);">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#" style="width: 175px; margin-right: 0px;">
+                <img class="logo" alt="MEETBOT LOGS" src="{{ url_for('static', filename='imgs/Fedora_logo.png') }}"
+                    style="width: 95%; height: auto;" />
+            </a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav me-auto pt-1">
+                    <li class="nav-item" id="loader">
+                        <a class="nav-link" aria-current="page" type="button">Recent meetings</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" aria-current="page" data-bs-toggle="modal" data-bs-target="#chanmode"
+                            type="button" onclick="populate_channel_list();">
+                            Meetings by Channel
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" type="button" href="#">
+                            Meetings by Group
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" type="button" href="#">
+                            About
+                        </a>
+                    </li>
+                </ul>
+                <div class="d-flex flex-column justify-content-center align-items-center">
+                    <div class="input-group">
+                        <input type="text" autocomplete="off" class="form-control" placeholder="Search meeting logs"
+                            aria-label="Search meeting logs" aria-describedby="findtext" style="width: 350px;"
+                            id="findtext">
+                    </div>
+                    <div class="input-group" data-bs-auto-close="true">
+                        <ul class="dropdown-menu w-100" id="dropdown">
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </nav>
+    <div class="container-lg">
+        <div class="card mt-1" id="bigcard">
+            <div class="card-body pt-0">
+                <div class="mt-4">
+                    <br><br>
+                    <article>
```

Comment:
I'm not sure why you choose to use the `article` tag here. A simple `div` should be more than enough.